### PR TITLE
Add bot.resources command

### PR DESCRIPTION
**Pull request by kingdom5500**
it's pretty self-explanatory.
image
the only think that needs to be noted is that the emotes don't work properly in this server for some reason, but they _probably_ will work on the main server

### CODE REVIEW

File: bot/constants.py [hunk 1]
```diff
@@ -49,6 +49,9 @@
 GREEN_CHEVRON = "<:greenchevron:418104310329769993>"
 RED_CHEVRON = "<:redchevron:418112778184818698>"
 WHITE_CHEVRON = "<:whitechevron:418110396973711363>"
+PAID_ICON = "<:paid_icon:432872213088305153>"
+FREEWARE_ICON = "<:freeware_icon:432872213088305153>"
+FREE_ICON = "<:free_icon:432872213088305153>"
```
Comment (shadycake) [inline bot/constants.py]:
```python
PAID_ICON = "<:paid_icon:432849192562458644>"`
FREEWARE_ICON = "<:freeware_icon:432848334143750144>"`
FREE_ICON = "<:free_icon:432864242715197441>"
```
I know you probably changed them for testing, but they gotta be right if we want to merge

File: bot/cogs/website.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                icon = PAYMENT_ICONS[payment]
+
+                lines.append(f"{icon} **[{name}]({url})**")
+                lines.append(f"{item_description}")
+
+        embed = Embed()
+        embed.title = "Useful Resources."
+        embed.url = resources_url
+
+        log.debug(f"{ctx.author} requested the resources list. Returning a paginated list.")
```
Comment (shadycake) [inline bot/cogs/website.py]: Consider logging the id, as this would only log the username which can change. Maybe add it in brackets after the username?

**Comment by gdude2002** [comment]
The format used for the JSON file has changed.

Where you'd get a single `url` before, each resource now has a `urls` attribute that is a list. It looks like this:

```json
"urls": [
  {
    "title": "Website",
    "url": "https://codecombat.com/",
    "icon": "regular/link"
  },
  {
    "title": "GitHub",
    "url": "https://github.com/codecombat/codecombat",
    "icon": "branding/github"
  }
],
```

The icons are font-awesome icons, and you can use any icons from font-awesome 5 (free or pro). A full list of these icons can be found [here](https://fontawesome.com/icons).

Please note that icons are given in the form `style/name`. Valid styles include `branding`, `regular`, `light` or `solid`. Font-awesome icons are varied based on the style you pick, so check which one you need from the icon list above!

File: bot/constants.py [hunk 3]
```diff
@@ -49,9 +49,10 @@
 GREEN_CHEVRON = "<:greenchevron:418104310329769993>"
 RED_CHEVRON = "<:redchevron:418112778184818698>"
 WHITE_CHEVRON = "<:whitechevron:418110396973711363>"
-PAID_ICON = "<:paid_icon:432872213088305153>"
-FREEWARE_ICON = "<:freeware_icon:432872213088305153>"
-FREE_ICON = "<:free_icon:432872213088305153>"
+PAID_ICON = "<:icon_paid:433750296724897798>"
+FREEWARE_ICON = "<:icon_freeware:433750375959494657>"
+FREE_ICON = "<:icon_free:433750340509368330>"
+LINK_ICON = "<:icon_link:433703566830469133>"
```
Comment (shadycake) [inline bot/constants.py]: They've changed again btw
```py
PAID_ICON = "<:icon_paid:433738703475834898>"
FREEWARE_ICON = "<:icon_freeware:433696378389004289>"
FREE_ICON = "<:icon_free:433696334457864201>"
LINK_ICON = "<:icon_link:433738861462421504>"
```

Comment (AnonGuy) [reply bot/constants.py]: Yeah, we're testing some stuff with different bot accounts. these emote ids are from the testing server, and they were pushed so I could test with a different token.

Comment (AnonGuy) [reply bot/constants.py]: This should be resolved once we wrap up testing and figure out what's wrong with the resources pagination.

**Comment by gdude2002** [comment]
You still have some review comments to address.

**Comment by 1mn** [comment]
This is being blocked by that weird pagination bug that we tried and failed to escalate to discord bughunt. I wonder if we can find a workaround somehow? Perhaps alter the parameters in such a way that the bug is no longer occurring? This is tricky..

**Comment by gdude2002** [comment]
**In order to expedite our current plans, we're requiring that all PRs be closed or merged.**

You have some review comments to address, and you've also got a conflict in `constants.py`.

**Comment by gdude2002** [comment]
Closing this as Juan wants to recreate it on GitLab

**Pull request closed without merging**